dev-server/mock/fagsak-1.json

```diff
@@ -90,7 +90,10 @@
                                 "beløp": 1054,
                                 "stønadFom": "2020-01-01"
                             }
-                        ]
+                        ],
+                        "resultat": "INNVILGET",
```

Comment:
Result and begrunnelse is moved to behandling level and can be removed from mock.